packages/calcite-components/src/components/text-area/text-area.tsx

```diff
@@ -462,6 +467,7 @@ export class TextArea
           }}
           cols={this.columns}
           disabled={this.disabled}
+          maxLength={this.limitText ? this.maxLength : -1}
```

Comment:
What does `-1` do here in this case?  Does that set `maxLength` to `undefined` as part of the Lumina compiler?

Reply:
It setting it to [`maxLength`'s initial value](https://developer.mozilla.org/en-US/docs/Web/API/HTMLTextAreaElement/maxLength#value).

Reply:
There we go, thank you for that link!  I couldn't seem to find docs on that when I was browsing that page just now.